app.go

```diff
@@ -39,38 +39,20 @@ func (a *App) Start(addr string) error {
 	if !strings.Contains(addr, ":") {
 		addr = fmt.Sprintf(":%s", addr)
 	}
-	fmt.Printf("Starting application at %s\n", addr)
+	a.Logger.WithField("interface", addr).Info("Starting application")
```

Comment:
Is there a reason you changed all of the logging?

Reply:
In particular, I'm curious about why the capitalization is different.

Reply:
Ended up adding a ton of logging to diagnose and the habit of only using the configured logger kicked in.  Have a personal nit when unstructured or inconsistent logging structures messages are emitted. :)

Capitalization changes are just from retyping -- not on purpose.

It's unrelated to issue -- I'll revert